docs/guide/device-and-platform-support.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Device & Platform Support"
+type: guide
+layout: docs
+parent_section: guide
+order: 5
+---
+
+Devices and platform support depends on how well the browsers support certain
+devices and APIs. A-Frame supports both flat (3D on a normal screen) and WebVR
+experiences.  Though its focus is heavily VR. We will break up support into two
+categories: flat and VR.
+
+<!--toc-->
+
+## Support for Flat Experiences
+
+Support for flat experiences primarily depends on the browsers' WebGL support.
+We can see what browsers support WebGL at [Can I use WebGL?][caniusewebgl].
+
+## Support for VR Experiences
+
+Support for VR experiences, along with WebGL support, depends on the browsers'
+WebVR API support. The browsers that support the WebVR API *at the moment you
+read this* depends on the browsers and not on A-Frame. As more and more
+browsers adopt the WebVR APIs, they will inherit support from A-Frame.
+
+You can see what browsers *currently* support the WebVR API at [Is WebVR
+Ready?][iswebvrready].
+
+This version of A-Frame supports these forms of the WebVR API:
+
+| WebVR API                     | Details                                                                             |
+|-------------------------------|-------------------------------------------------------------------------------------|
+| [WebVR 1.0][webvr-1.0]        | Primary support.                                                                    |
+| Deprecated `getVRDevices` API | Supported through the [WebVR Polyfill][webvr-polyfill].                             |
```

Comment:
I wish it was true, but unfortunately the WebVR Polyfill does not polyfill `navigator.getVRDevices`, only `navigator.getVRDisplays` if it's not available. it doesn't fall back. if you set `WebVRConfig. ENABLE_DEPRECATED_API = true`, then `navigator.getVRDevices` will be polyfilled, but `navigator.getVRDisplays` won't.

we could add conditional logic to `src/index.js` (though Boris says he wants to remove the option very soon):

```diff
diff --git a/src/index.js b/src/index.js
index c704b48..b438546 100644
--- a/src/index.js
+++ b/src/index.js
@@ -6,6 +6,7 @@ window.hasNativeWebVRImplementation = !!navigator.getVRDisplays || !!navigator.g
 
 window.WebVRConfig = window.WebVRConfig || {
   CARDBOARD_UI_DISABLED: true,
+  ENABLE_DEPRECATED_API: navigator.getVRDevices && !navigator.getVRDisplays
   ROTATE_INSTRUCTIONS_DISABLED: true,
   TOUCH_PANNER_DISABLED: true,
   MOUSE_KEYBOARD_CONTROLS_DISABLED: true,
```